src/test/java/com/hocs/test/pages/give_me_a_case/fetch.java

```diff
@@ -43,6 +50,69 @@
 
     Workstacks workstacks;
 
+    Page page;
+
+    MarkUpDecision markUpDecision;
+
+    WebDriver driver;
+
+    public void getFirstUnallocatedMINCaseDataInputCase() {
+        WebElementFacade firstUnallocatedMINCase = findAll(
+                "//td[following-sibling::td[1][contains(text(), 'Data "
+                        + "Input')]][following-sibling::td[2]"
+                        + "[not(contains(text(), '@'))]][descendant::a[contains(text(), 'MIN')]]").get(0);
+        firstUnallocatedMINCase.click();
+        page.clickOn(workstacks.allocateToMeButton);
```

Comment:
Eg here `clickOn(workstacks.allocateToMeButton);` will work without `Page page` and `page.`